docs/dialect/graphblas_dialect_tutorials/graphblas_lower/graphblas_matrix_multiply.ipynb

```diff
@@ -0,0 +1,873 @@
+    "%cf4 = constant 4.0 : f64\n",
+    "%answer = graphblas.matrix_multiply %argA, %argB { semiring = \"plus_times\" } : (tensor<?x?xf64, #CSR64>, tensor<?x?xf64, #CSC64>) to tensor<?x?xf64, #CSR64> {\n",
+    "        ^bb0(%value: f64):\n",
+    "            %result = std.addf %value, %cf4: f64\n",
+    "            graphblas.yield %result : f64\n",
+    "    }\n",
+    "```\n",
+    "The NumPy equivalent of this code would be `answer = (argA @ argB) + 4.0`.\n",
+    "\n",
+    "The region specifies element-wise post-processing done on values that survived the masking (applies to all elements if no mask). We'll go into deeper details later on on how to write a region using `graphblas.yield`. \n",
+    "\n",
+    "Let's create some example input matrices."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "d263eb46",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "indices = np.array(\n",
+    "    [\n",
+    "        [0, 3],\n",
+    "        [1, 3],\n",
+    "        [2, 0],\n",
+    "        [3, 0],\n",
+    "        [3, 1],\n",
+    "    ],\n",
+    "    dtype=np.uint64,\n",
+    ")\n",
+    "values = np.array([1, 2, 3, 4, 5], dtype=np.float64)\n",
+    "sizes = np.array([4, 4], dtype=np.uint64)\n",
+    "sparsity = np.array([False, True], dtype=np.bool8)\n",
+    "\n",
+    "A = mlir_graphblas.sparse_utils.MLIRSparseTensor(indices, values, sizes, sparsity)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "c6242a8b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "indices = np.array(\n",
+    "    [\n",
+    "        [0, 1],\n",
+    "        [0, 3],\n",
+    "        [1, 1],\n",
+    "        [1, 3],\n",
+    "        [2, 0],\n",
+    "        [2, 2],\n",
+    "        [3, 0],\n",
+    "        [3, 2],\n",
+    "    ],\n",
+    "    dtype=np.uint64,\n",
+    ")\n",
+    "values = np.array([1, 2, 3, 4, 5, 6, 7, 8], dtype=np.float64)\n",
+    "sizes = np.array([4, 4], dtype=np.uint64)\n",
+    "sparsity = np.array([False, True], dtype=np.bool8)\n",
+    "\n",
+    "B_csr = mlir_graphblas.sparse_utils.MLIRSparseTensor(indices, values, sizes, sparsity)\n",
+    "B = engine.convert_csr_to_csc(B_csr)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "849c050e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "indices = np.array(\n",
+    "    [\n",
+    "        [0, 1],\n",
+    "        [0, 2],\n",
+    "        [1, 1],\n",
+    "        [1, 2],\n",
+    "        [2, 1],\n",
+    "        [2, 2],\n",
+    "        [3, 1],\n",
+    "        [3, 2],\n",
+    "    ],\n",
+    "    dtype=np.uint64,\n",
+    ")\n",
+    "values = np.array([-0.1, 0.2, -0.3, 0.4, -0.5, 0.6, -0.7, 0.8], dtype=np.float64)\n",
+    "sizes = np.array([4, 4], dtype=np.uint64)\n",
+    "sparsity = np.array([False, True], dtype=np.bool8)\n",
+    "\n",
+    "mask = mlir_graphblas.sparse_utils.MLIRSparseTensor(indices, values, sizes, sparsity)"
```

Comment:
That's a great idea. https://github.com/metagraph-dev/mlir-graphblas/pull/85/commits/ba76211656fa5079681aa44aaa78908234bbdd9f implements this suggestion. I kept the dtype as `np.float64` since we don't support boolean sparse tensors yet.